.clang-format

```diff
@@ -92,6 +94,7 @@ MacroBlockBegin: ''
 MacroBlockEnd: ''
 MaxEmptyLinesToKeep: 1
 NamespaceIndentation: All
+PackConstructorInitializers: BinPack
```

Comment:
The example on https://clang.llvm.org/docs/ClangFormatStyleOptions.html doesn't make it seem like `BinPack` is what we want, but it doesn't seem to format it like that. `NextLine` also doesn't create new changes, but the example looks like what we have currently, so maybe we should use that instead.

Reply:
Oops, I misread the option. I agree that `NextLine` is what we want.